src/components/ui-base/Button/button.module.css

```diff
@@ -0,0 +1,34 @@
+.button {
+    color: var(--amp-colors-button-text);
+    background-color: var(--amp-colors-button-primary);
+    border-radius: var(--amp-default-border-radius);
+
+    border: 1px solid;
+    border-color: inherit;
+    height: 2.5rem;
+    width: 100%;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5rem; /* 150% */
+    padding: 0 1rem;
+    transition: all .1s ease;
+}
+
+.buttonError {
+    border: 1px solid var(--amp-colors-error-border);
+}
+
+.button:focus:enabled {
+    border: 2px solid var(--amp-colors-accent-primary);
+    outline: none;
+}
+
+.button:disabled {
+    background-color: var(--amp-colors-button-primary-disabled);
+    cursor: not-allowed;
+}
+
+/* Additional styling to support screen readers */
+button[aria-expanded="true"] {
+    background-color: #004080;  /* Can visually indicate button state */
```

Comment:
Would be good to export this as a var too

Reply:
same as above.